wiki/Beatmapping/Overmapping/en.md

```diff
@@ -1,9 +1,15 @@
----
-stub: true
----
-
 # Overmapping
 
-**Overmapping** is a term used when [hit objects](/wiki/Gameplay/Hit_object) or arrangement of hit objects are mapped in a way that makes the [difficulty](/wiki/Beatmap/Difficulty) of the beatmap contrast the song.
+In beatmapping, **overmapping** is a mapping technique where [hit objects](/wiki/Gameplay/Hit_object) are deliberately placed at a much higher contrast compared to what the song (or the rest of the map) conceivably calls for.
+
+This is mainly done by mapping to inaudible or non-existant sounds (i.e. adding notes on blue ticks when the song only uses red ticks). Less frequently, overmapping can also be achieved by adding exorbitantly large [jumps](/wiki/Beatmap/Pattern/osu!/Jump) or sliders with very high [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity).
+
+## Examples
+
+![Example of overmapping by excessively high object density](img/overmap-stream.gif "Example of overmapping by excessively high object density")
+
+![Example of overmapping by excessively high object spacing](img/overmap-jump.gif "Example of overmapping by excessively high object spacing")
+
+![Example of overmapping by excessively high slider velocity](img/overmap-slider.gif "Example of overmapping by excessively high slider velocity")
 
-This is usually done by adding unnecessarily large [jumps](/wiki/Beatmap/Pattern/osu!/Jump), [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) changes, or by mapping to inaudible or non-existant sounds (i.e. adding notes on blue ticks when the song uses red ticks).
+Please note that while overmapping is generally discouraged, this technique remains perfectly acceptable to use when applied under an appropriate context.
```

Comment:
i wonder if this can be worded in such a way that it doesn't sound like a vague opinion from the author, perhaps by adding concrete examples

```suggestion
While overmapping is generally discouraged, there are contexts where the technique is acceptable.
```

Reply:
yeah